gradle/src/main/groovy/com/flurry/android/symbols/NdkSymbolUpload.groovy

```diff
@@ -60,15 +61,10 @@ class NdkSymbolUpload {
             }
         }
 
-        try {
-            variant.getExternalNativeBuildProviders().each {
-                it.configure() {
-                    searchForSharedObjectFiles(it)
-                }
+        variant.getExternalNativeBuildProviders().each {
```

Comment:
was the bug related to catching/not catching this? If not, should we be changing this?

Reply:
more generally, what is the bug this commit is fixing?

Reply:
This was a warning message; which was confusing me while debugging. This old interface is to support AGP versions older than com.android.tools.build:gradle:3.3.1 which we are not supporting as the minimum is 4.2 so safe to remove. See: https://github.com/firebase/firebase-android-sdk/issues/198#issuecomment-461408446